personal_assistant/pa_note/templates/pa_note/add_note.html

```diff
@@ -0,0 +1,11 @@
+{% extends "pa_core/home.html" %}
+
+{% block content %}
+    <h1>Add note</h1>
+    <form method="post">
+        {% csrf_token %}
+        {{ form.as_p }}
+        <button type="submit">Save</button>
+    </form>
+    <a href="{% url 'pa_note:note' %}">Back to notes list</a>
+{% endblock %}
```

Comment:
Дещо скоротиться файл, якщо унаслідуватися від іншого:
```suggestion
{% extends 'pa_core/form.html' %}

{% block title %}Add note{% endblock %}

{% block form %}{{ form.as_p }}{% endblock %}

{% block button %}Save{% endblock %}

```